src/components/Toggle/Toggle.md

```diff
@@ -0,0 +1,24 @@
+# Toggle
+Allows for turning on or off a setting.
```

Comment:
I would mention that this is best suited for simple, singular configuration (e.g. application level settings) whereas the checkbox is better suited when there is a longer list of individual choices or in a form (e.g. signing up for different newsletters during registration for an event).